yt_dlp/extractor/kick.py

```diff
@@ -69,36 +92,39 @@ def _real_extract(self, url):
                 response, ('livestream', ('session_title', 'slug')), get_all=False, default=''),
             'description': traverse_obj(response, ('user', 'bio')),
             'channel': channel,
-            'channel_id': str_or_none(traverse_obj(response, 'id', ('livestream', 'channel_id'))),
+            'channel_id': int_or_none(traverse_obj(response, 'id', ('livestream', 'channel_id'))),
             'uploader': traverse_obj(response, 'name', ('user', 'username')),
-            'uploader_id': str_or_none(traverse_obj(response, 'user_id', ('user', 'id'))),
+            'uploader_id': int_or_none(traverse_obj(response, 'user_id', ('user', 'id'))),
```

Comment:
revert these. `uploader_id` and `channel_id` values need to be str

Reply:
Fixed : https://github.com/yt-dlp/yt-dlp/pull/10572/commits/48dd2e157b46b2acf310fc0a58736c2883118b04